package/westeros-sink/0003-westeros-close-only-new-connection.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/westeros-sink/westeros-sink.c b/westeros-sink/westeros-sink.c
```

Comment:
Should we make a seperate PR out of this to sink the westeros patches? I'd assume other packages will benefit from that as well?

Reply:
Yes created